readme.rst

```diff
@@ -315,6 +329,13 @@ This will create the required software list (``.sphinx/requirements.txt``)
 which is used to create a virtual environment (``.sphinx/venv``) and install
 dependency software within it.
 
+To install the validation tools explicitly:
```

Comment:
```suggestion
To install the validation tools:
```
You probably added the "explicitly" to indicate that they are also installed when you run the checks for the first time, but it reads a bit strange to me. We can just leave it out imo.